Unable to build v3.20.2

Hello 👋🏼 

Just saw the latest version and decided to update the Arch package. However hit this:

```
   Compiling bacon v3.20.2 (/build/bacon/src/bacon)
error[E0432]: unresolved import `rodio::OutputStream`
   --> src/sound/play_sound.rs:3:5
    |
  3 |     rodio::OutputStream,
    |     ^^^^^^^^^^^^^^^^^^^ no `OutputStream` in the root
    |
note: found an item that was configured out
   --> /build/.cargo/registry/src/index.crates.io-1949cf8c6b5b557f/rodio-0.21.1/src/lib.rs:194:31
    |
193 | #[cfg(feature = "playback")]
    |       -------------------- the item is gated behind the `playback` feature
194 | pub use crate::stream::{play, OutputStream, OutputStreamBuilder, PlayError, StreamError};
    |                               ^^^^^^^^^^^^

error[E0412]: cannot find type `StreamError` in crate `rodio`
   --> src/sound/play_sound.rs:97:24
    |
 97 |     RodioStream(rodio::StreamError),
    |                        ^^^^^^^^^^^ not found in `rodio`
    |
note: found an item that was configured out
   --> /build/.cargo/registry/src/index.crates.io-1949cf8c6b5b557f/rodio-0.21.1/src/lib.rs:194:77
    |
193 | #[cfg(feature = "playback")]
    |       -------------------- the item is gated behind the `playback` feature
194 | pub use crate::stream::{play, OutputStream, OutputStreamBuilder, PlayError, StreamError};
    |                                                                             ^^^^^^^^^^^

error[E0412]: cannot find type `PlayError` in crate `rodio`
   --> src/sound/play_sound.rs:98:22
    |
 98 |     RodioPlay(rodio::PlayError),
    |                      ^^^^^^^^^ not found in `rodio`
    |
note: found an item that was configured out
   --> /build/.cargo/registry/src/index.crates.io-1949cf8c6b5b557f/rodio-0.21.1/src/lib.rs:194:66
    |
193 | #[cfg(feature = "playback")]
    |       -------------------- the item is gated behind the `playback` feature
194 | pub use crate::stream::{play, OutputStream, OutputStreamBuilder, PlayError, StreamError};
    |                                                                  ^^^^^^^^^

error[E0412]: cannot find type `StreamError` in crate `rodio`
   --> src/sound/play_sound.rs:100:18
    |
100 | impl From<rodio::StreamError> for SoundError {
    |                  ^^^^^^^^^^^ not found in `rodio`
    |
note: found an item that was configured out
   --> /build/.cargo/registry/src/index.crates.io-1949cf8c6b5b557f/rodio-0.21.1/src/lib.rs:194:77
    |
193 | #[cfg(feature = "playback")]
    |       -------------------- the item is gated behind the `playback` feature
194 | pub use crate::stream::{play, OutputStream, OutputStreamBuilder, PlayError, StreamError};
    |                                                                             ^^^^^^^^^^^

error[E0412]: cannot find type `StreamError` in crate `rodio`
   --> src/sound/play_sound.rs:101:23
    |
101 |     fn from(e: rodio::StreamError) -> Self {
    |                       ^^^^^^^^^^^ not found in `rodio`
    |
note: found an item that was configured out
   --> /build/.cargo/registry/src/index.crates.io-1949cf8c6b5b557f/rodio-0.21.1/src/lib.rs:194:77
    |
193 | #[cfg(feature = "playback")]
    |       -------------------- the item is gated behind the `playback` feature
194 | pub use crate::stream::{play, OutputStream, OutputStreamBuilder, PlayError, StreamError};
    |                                                                             ^^^^^^^^^^^

error[E0412]: cannot find type `PlayError` in crate `rodio`
   --> src/sound/play_sound.rs:105:18
    |
105 | impl From<rodio::PlayError> for SoundError {
    |                  ^^^^^^^^^ not found in `rodio`
    |
note: found an item that was configured out
   --> /build/.cargo/registry/src/index.crates.io-1949cf8c6b5b557f/rodio-0.21.1/src/lib.rs:194:66
    |
193 | #[cfg(feature = "playback")]
    |       -------------------- the item is gated behind the `playback` feature
194 | pub use crate::stream::{play, OutputStream, OutputStreamBuilder, PlayError, StreamError};
    |                                                                  ^^^^^^^^^

error[E0412]: cannot find type `PlayError` in crate `rodio`
   --> src/sound/play_sound.rs:106:23
    |
106 |     fn from(e: rodio::PlayError) -> Self {
    |                       ^^^^^^^^^ not found in `rodio`
    |
note: found an item that was configured out
   --> /build/.cargo/registry/src/index.crates.io-1949cf8c6b5b557f/rodio-0.21.1/src/lib.rs:194:66
    |
193 | #[cfg(feature = "playback")]
    |       -------------------- the item is gated behind the `playback` feature
194 | pub use crate::stream::{play, OutputStream, OutputStreamBuilder, PlayError, StreamError};
    |                                                                  ^^^^^^^^^

Some errors have detailed explanations: E0412, E0432.
For more information about an error, try `rustc --explain E0412`.
error: could not compile `bacon` (lib) due to 7 previous errors
warning: build failed, waiting for other jobs to finish...
```

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -34,7 +34,7 @@ lazy-regex = "3.4.1"
 notify = "8.2.0"
 paste = "1.0"
 pretty_assertions = "1.4"
-rodio = { version = "0.21", optional = true, default-features = false, features = [ "mp3" ] }
+rodio = { version = "0.21", optional = true, default-features = false, features = [ "mp3", "playback" ] }
 rustc-hash = "2"
 serde = { version = "1.0", features = ["derive"] }
 serde_json = "1.0"

--- a/src/sound/play_sound.rs
+++ b/src/sound/play_sound.rs
@@ -1,6 +1,6 @@
 use {
     super::*,
-    rodio::OutputStream,
+    rodio::OutputStreamBuilder,
     std::{
         fmt,
         io::Cursor,
@@ -130,9 +130,9 @@ pub fn play_sound(
 ) -> Result<(), SoundError> {
     debug!("play sound: {:#?}", psc);
     let Sound { bytes, duration } = get_sound(psc.name.as_deref())?;
-    let (_stream, stream_handle) = OutputStream::try_default()?;
+    let stream = OutputStreamBuilder::open_default_stream()?;
     let sound = Cursor::new(bytes);
-    let sink = stream_handle.play_once(sound)?;
+    let sink = rodio::play(stream.mixer(), sound)?;
     sink.set_volume(psc.volume.as_part());
     if interrupt.recv_timeout(duration).is_ok() {
         info!("sound interrupted");

